fe/be-java-extensions/paimon-scanner/src/main/java/org/apache/doris/paimon/PaimonJniScanner.java

```diff
@@ -87,9 +89,18 @@ public PaimonJniScanner(int batchSize, Map<String, String> params) {
 
     @Override
     public void open() throws IOException {
-        initTable();
-        initReader();
-        resetDatetimeV2Precision();
+        try {
+            Thread.currentThread().setContextClassLoader(classLoader);
```

Comment:
Add comment in code to explain why need class loader

Reply:
done